src/Framework/App.Ref/src/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -214,21 +214,19 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           DependsOnTargets="_ResolveTargetingPackContent"
           Inputs="@(RefPackContent)"
           Outputs="@(RefPackContent->'$(LayoutTargetDir)%(PackagePath)%(RecursiveDir)%(FileName)%(Extension)')">
+    <RemoveDir Directories="$(TargetingPackLayoutRoot)" />
     <Copy SourceFiles="@(RefPackContent)"
           DestinationFiles="@(RefPackContent->'$(LayoutTargetDir)%(PackagePath)%(RecursiveDir)%(FileName)%(Extension)')"
           UseHardlinksIfPossible="true" />
     <Message Importance="High" Text="$(MSBuildProjectName) -> $(LayoutTargetDir)" />
   </Target>
 
-  <ItemGroup>
-    <CreateDirectory Include="$(LocalInstallationOutputPath)" />
```

Comment:
@dsplaisted @wli3 is it a bug that the .NET SDK will not download targeting pack if the corresponding .dotnet/packs/Microsoft.AspNetCore.App.Ref/ directory exists❔ This line led me on a wild goose chase in 6.0.1 part of Servicing Readiness Exercise because it seemed nothing would download our 6.0.0 targeting pack☹️

Reply:
If I'm understanding correctly that's what's supposed to happen.  If the targeting pack is already installed locally it won't download it.

Reply:
@dsplaisted you're understanding my question and I agree avoiding repeated downloads of the same packs would be wasteful.

My surprise was specifically about this _empty_ directory case. Guess I expected something more complicated or thorough.